lib/vimaly/client.rb

```diff
@@ -86,7 +86,11 @@ def bin(name_or_regex)
 
     def bins
       @bins ||= begin
-        get('/bins?max-results=500').map do |bin_data|
+        per_page = 500
+        data = (0..1).inject([]) do |list, page|
+          list += get("/bins?max-results=#{per_page}&page-token=#{page * 500}")
```

Comment:
LGTM, their API is a bit odd…`page-token` what does that mean?

Assuming it works.